diff --git a/bazel/cc_toolchains/cc_toolchain_optimization.bzl b/bazel/cc_toolchains/cc_toolchain_optimization.bzl
index c0d83d07..7d1aded4 100644
--- a/bazel/cc_toolchains/cc_toolchain_optimization.bzl
+++ b/bazel/cc_toolchains/cc_toolchain_optimization.bzl
@@ -51,7 +51,7 @@ cpu_flags = feature(
     flag_sets = [
         flag_set(
             actions = all_compile_actions + all_link_actions,
-            flag_groups = [flag_group(flags = ["-march=armv8.2-a"])],
+            flag_groups = [flag_group(flags = ["-march=armv8-a"])],
             with_features = [with_feature_set(["aarch64_target"])],
         ),
         flag_set(
