diff --git a/bazel/cc_toolchains/cc_toolchain_optimization.bzl b/bazel/cc_toolchains/cc_toolchain_optimization.bzl
index 13e10314..8ad28d84 100644
--- a/bazel/cc_toolchains/cc_toolchain_optimization.bzl
+++ b/bazel/cc_toolchains/cc_toolchain_optimization.bzl
@@ -46,7 +46,7 @@ cpu_flags = feature(
     flag_sets = [
         flag_set(
             actions = ACTION_NAME_GROUPS.all_cc_compile_actions + ACTION_NAME_GROUPS.all_cc_link_actions,
-            flag_groups = [flag_group(flags = ["-march=armv8.2-a"])],
+            flag_groups = [flag_group(flags = ["-march=armv8-a"])],
             with_features = [with_feature_set(["aarch64_target"])],
         ),
         flag_set(
